packages/story-editor/src/components/panels/document/taxonomies/HierarchicalTermSelector.js

```diff
@@ -124,7 +128,7 @@ function HierarchicalTermSelector({ noParentId = NO_PARENT_VALUE, taxonomy }) {
     );
 
   const categories = useMemo(() => {
-    if (termCache[taxonomy.restBase]) {
+    if (termCache?.[taxonomy.restBase]) {
```

Comment:
using optional chaining here so that unit tests don't need all of these props to run